L1Trigger/L1CaloTrigger/plugins/L1EGammaCrystalsEmulatorProducer.cc

```diff
@@ -1176,28 +1176,45 @@ void L1EGCrystalClusterEmulatorProducer::produce(edm::Event& iEvent, const edm::
 }
 
 bool L1EGCrystalClusterEmulatorProducer::passes_iso(float pt, float iso) {
+  bool is_iso = true;
   if (pt < slideIsoPtThreshold) {
     if (!((a0_80 - a1_80 * pt) > iso))
-      return false;
+      is_iso = false;
   } else {
     if (iso > a0)
-      return false;
+      is_iso = false;
   }
-  return true;
+  if (pt > 130)
```

Comment:
please define a clearly named constant and use it in place of magic number 130 everywhere